### ISSUE TITLE
Poor performance on KITTI Test Dataset

### ISSUE THREAD

**Issue by JiashengWan** [dragonfly606/MonoCD#10]
Hi, I tried to submit my test results to KITTI dataset but the performance is very poor

Car (Detection) | 32.79 % | 20.77 % | 17.11 %
Car (Orientation) | 29.06 % | 17.75 % | 14.62 %
Car (3D Detection) | 0.13 % | 0.04 % | 0.05 %
Car (Bird's Eye View) | 0.67 % | 0.46 % | 0.34 %

The main difference I made from train to test is 

# Training/Validation (monocd_train.yaml):
TEST: ("kitti_train",)    
TEST_SPLIT: "val"        # Uses validation split from training data

# Testing (monocd_test.yaml):
TEST: ("kitti_test",)     
TEST_SPLIT: "test"       # Uses separate test set

# Originally both configs had:
UNCERTAINTY_AS_CONFIDENCE: True

# We changed test config to:
UNCERTAINTY_AS_CONFIDENCE: False   # Disabled for test to avoid zero confidence scores

The main issue seem to be that training and validation performs well but the test performance is very poor

Any idea what are the settings I need to change before I submit to KITTI?

**Comment by dragonfly606** [comment]
It seems that, in addition to the bad 3D detection results, the 2D detection results are not very good too. Maybe you encountered a training crash. My suggestion is to train multiple times to avoid accidents. It is also recommended to set TRAIN_SPLIT: "trainval".
